src/main/features/response/form/models/whenDidYouPay.ts

```diff
@@ -0,0 +1,56 @@
+import { IsDefined, MaxLength, ValidateNested } from 'class-validator'
+import { IsPastDate } from 'forms/validation/validators/datePastConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { IsValidYearFormat } from 'app/forms/validation/validators/isValidYearFormat'
+import { MomentFactory } from 'common/momentFactory'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+import { MomentFormatter } from 'app/utils/momentFormatter'
+
+export class ValidationErrors {
+  static readonly EXPLANATION_REQUIRED: string = 'Explain how did you pay the amount claimed'
+  static readonly DATE_REQUIRED: string = 'Enter a date'
```

Comment:
this value is common to other places please use the global validation errors message.
REQ

Reply:
I though about it but did not recommend it at the time because there is not common constant for that message at the moment and I did not want to trigger more changes. Agree that it is good to have however I would recommend making that change (extracting it in all places where date is validated) in next PR to review it properly.